.evergreen/resync-specs.sh

```diff
@@ -134,7 +134,7 @@ do
     uri|uri*options)
       cpjson uri-options/tests uri_options
       ;;
-    versioned-api)
+    stable-api)
       cpjson versioned-api/tests versioned-api
```

Comment:
Should we wait until the spec changes get merged so that we can update this line as well?

Reply:
Sounds good to me.

Reply:
Let's just remove this change and merge.